hls4ml/backends/vitis/vitis_backend.py

```diff
@@ -0,0 +1,42 @@
+import os
+import sys
+
+from hls4ml.backends import VivadoBackend
+from hls4ml.model.flow import register_flow, get_flow
+from hls4ml.report import parse_vivado_report
+
+
+class VitisBackend(VivadoBackend):
```

Comment:
I have some concerns inheriting from the VivadoBackend for two reasons. On is that it seems to violate the "is a" requirement for inheritance. Maybe the common features can be factored out? The other is more long-term. In the future when Vitis HLS becomes our primary backend, do we still want to inherit from VivadoBackend?

Reply:
I would agree that having a parent `XilinxBackend` from which `VivadoBackend` and `VitisBackend` are derived would be more elegant, but if we plan on deprecating and ultimately phasing out one and switching to the other I think this is fine. The clean solution would require another level in the hierarchy of backend inheritance, which may confuse the developers of flows. Once we are happy with the state of Vitis (i.e., when we gather more feedback from users, which seems very positive so far), we can move stuff from `VivadoBackend` to `VitisBackend` and make the Vivado the "override" of Vitis.